### DISCUSSION TITLE
How to join Organization

### DISCUSSION THREAD

**Discussion by itsdotnet** [community/community#70618] [Other Feature Feedback, Questions, & Ideas]
Is it possible to join organization if owers didn't invite to join.
What should i do if I want to join Organizations.

**Comment by tuves** [comment]
Welcome to the GitHub Community, @itsdotnet, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by ghostinhershell** [comment]
Hey @itsdotnet, 

Thanks so much for reaching out. You will need to reach out to the owners of the organization so that they can invite you to join their organization. You can learn more here:[ Inviting users to join your organization](https://docs.github.com/en/organizations/managing-membership-in-your-organization/inviting-users-to-join-your-organization).
